### PR TITLE
EOH fix & change

### PR DESCRIPTION
**Pull request by HoleFish**
Fix https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/issues/16279 (not tested)
Fix https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/issues/16585
Change astral array EU formula again. (slight simplify)
Ability to insert planet block by rightclicking the controller with planet block in hand.
Limit astral array amount up to 10000.

### CODE REVIEW

**Comment by Persus1** [comment]
the changed power formula for astral arrays looks like a bit to overpowered or am i missing something?

With maxed eoh:
- T0 planets need around 100,5% of the previous power required
- T5 planets need around 97,64% of the previous power required
- T9+ planets need around 79,62% of the previous power required

while the earlier planets work just fine T9+ planets get a massive power reduction of over 20% which seems like a pretty noticeable buff.
(example of a 1024 parallel eoh)
image

**Comment by HoleFish** [comment]
oh that's a tooltip mistake

File: src/main/java/com/github/technus/tectech/thing/metaTileEntity/multi/GT_MetaTileEntity_EM_EyeOfHarmony.java [hunk 1]
```diff
@@ -1048,12 +1072,26 @@ public GT_Multiblock_Tooltip_Builder createTooltip() {
                 "This multiblock can perform parallel processing by placing Astral Array Fabricators into the input bus.")
             .addInfo(
                 "They are stored internally and can be retrieved via right-clicking the controller with a wire cutter.")
-            .addInfo("The amount of parallel is calculated via these formulas:")
-            .addInfo(GREEN + "Parallel exponent = floor(log(8 * Astral Array amount) / log(1.7))")
+            .addInfo(
+                "The maximum amount of Astral Array is " + formatNumbers(ASTRAL_ARRAY_LIMIT)
```
Comment (GDCloudstrike) [inline src/main/java/com/github/technus/tectech/thing/metaTileEntity/multi/GT_MetaTileEntity_EM_EyeOfHarmony.java:1076]: nit: "The maximum amount of stored Astral Arrays is " would be better here

**Review by GDCloudstrike — CHANGES_REQUESTED** [review]
Thanks for the tierskip fix and adding the planet block functionality, that's a really nice qol change. 
About the astral array limit, what is the reason for implementing such a limit? I'm not against it existing (as you wont ever reach 10k astral arrays anyways), it's just a surprising change to see. 
A small nit regarding the limit number, if you want to keep it around the 10k range, I'd suggest changing it to 8637 instead of the flat 10k, as that is the closest 'step' in astral array amounts that grants more parallel, else you'd just have up to 1363 astral arrays sitting there doing nothing.

**Comment by HoleFish** [comment]
> what is the reason for implementing such a limit?

Ive seen some crazy players trying inserting infinite astral arrays which caused parallel overflow :trollface:

**Review by GDCloudstrike — APPROVED** [review]

**Pull request merged**